powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/database/entity/Fido2AuthenticatorEntity.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2024 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.powerauth.fido2.database.entity;
+
+import com.wultra.security.powerauth.client.model.enumeration.SignatureType;
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+/**
+ * Entity representing a FIDO2 Authenticator details.
+ *
+ * @author Jan Pesek, jan.pesek@wultra.com
+ */
+@Entity
+@Table(name = "pa_fido2_authenticator")
+@Data
```

Comment:
The `@Data` annotation produces `equals` and `hashCode` implementations which are not suitable for JPA classes. Can we add our own implementations?

Reply:
Thanks. Now both `Fido2Authenticator` and `Fido2AuthenticatorEntity` has `equals` and `hashCode` based on `aaguid` only.